src/markdown-pages/install-with-kurl/migrating-csi.md

```diff
@@ -73,3 +73,92 @@ kURL does the following when you use the specs above to migrate data from Rook:
+
+#### OpenEBS
+
+As stated in the [OpenEBS official documentation](https://openebs.io/docs/additional-info/faqs#:~:text=You%20can%20run%20these%20using,nodes%20in%20the%20Kubernetes%20cluster), OpenEBS requires 2G of memory and 2 CPUs, so it is crucial to ensure the cluster has these spare resources available prior to initiating the migration. It is crucial to be aware that OpenEBS and Longhorn both utilize the node's filesystem for data storage, therefore, it is essential to ensure that the filesystem is adequately sized to accommodate twice the amount of data stored by Longhorn.
+
+### Preparation
+
+To ensure a smooth migration process, it is important to take certain steps before starting it. This includes identifying and addressing the most common causes for a migration failure. By doing so, you can minimize the risk of encountering issues during the migration and increase the chances of success.
+
+#### Check Longhorn Volumes health
+
+It is crucial to ensure that the volumes being migrated are in a healthy state before proceeding. The migration won’t proceed if the volumes are not healthy. You can check the status of the volumes using the provided command below. If any volumes are reported as not healthy in the "Robustness" column, it is recommended to address the issue before proceeding with the migration.
+
+```
+$ kubectl get volumes.longhorn.io -A
+```
+
+In many cases, volume health can be attributed to issues with volume replication. Specifically, when multiple replicas are configured for a volume but not all of them have been scheduled yet. To gain more insight, you can review the individual volumes by editing them.
+
+```
+$ kubectl edit volumes.longhorn.io -n longhorn-system <volume name>
+```
+
+_During the migration process, when running on a single node cluster, the system will automatically scale down the number of replicas to 1 in all Longhorn volumes as a measure to ensure the volumes are in a healthy state before beginning the data transfer. This is done to minimize the risk of a migration failure._
+
+#### Check Longhorn Nodes health
+
+To ensure a successful migration, it is crucial to verify that all Longhorn nodes are in a healthy state: able to handle scheduled workloads, and not over-provisioned. You can check the status of the Longhorn nodes by running the following command:
+
+```
+$ kubectl get nodes.longhorn.io -A
+```
+
+If any node is not reported as "Ready" and "Schedulable", you can obtain more information by individually editing the node and checking its "Status" property.
```

Comment:
once again do you mean to suggest edit? would it be enough to describe node or get node -oyaml?